api/stse_device_management.c

```diff
@@ -170,6 +170,7 @@ stse_ReturnCode_t stse_device_unlock(stse_Handler_t *pSTSE, PLAT_UI8 *pPassword,
 {
```

Comment:
Changes are small enough to guarantee the non regression, the modification is ok for me.